contracts/test/foundry/FleekERC721/AccessPoints/AccessPointsAutoApprovalOn.t.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.17;
+
+import "../TestBase.sol";
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+import {FleekAccessControl} from "contracts/FleekAccessControl.sol";
+import "../../../../contracts/FleekERC721.sol";
+
+contract Test_FleekERC721_AccessPoint is Test_FleekERC721_Base {
+    using Strings for address;
+    uint256 internal tokenId;
+
+    function assertAccessPointJSON(
```

Comment:
Since we have 2 AP test files, should we create a base for AP test and move this into there so it could be shared?